src/MarginModels/PointSetMarginModel.py

```diff
@@ -65,14 +64,12 @@ def _handleInput(self, paramInput):
     super()._handleInput(paramInput)
     for child in paramInput.subparts:
       if child.getName() == 'failedDataFileID':
-        self.failedDataFileID = self.setVariable(child.value)
-        #self._variableDict['failedDataFileID'] = self.failedDataFileID
+        self.setVariable('failedDataFileID', child.value)
       if child.getName() == 'marginID':
-        self.marginID = self.setVariable(child.value)
-        #self._variableDict['marginID'] = self.marginID
+        self.setVariable('marginID', child.value)
```

Comment:
Also here, do not use setVariable method, just use 
```python
self.marginID = child.value
```

Reply:
same as above, both have been edited